trymito.io/pages/teams.tsx

```diff
@@ -0,0 +1,215 @@
+                    <div className={iconAndTextCardStyles.icon_and_text_card + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={iconAndTextCardStyles.icon}>
+                            <Image className={iconAndTextCardStyles.icon} src={PrivateIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Completely <br/> Private
+                        </h1>
+                        <p>
+                            The Mito Pro and Enterprise plans are totally local. No data leaves your computer, ever. Mito is designed for your security first. 
+                        </p>
+                    </div>
+                </div>
+                <div className={pageStyles.subsection}>
+                    <div className={iconAndTextCardStyles.icon_and_text_card}>
+                        <div className={iconAndTextCardStyles.icon}>
+                            <Image className={iconAndTextCardStyles.icon} src={DocsIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Practical <br/> Documentation
+                        </h1>
+                        <p>
+                            Our documentation is best-in-class, so you can get started without complex trainings.
+                        </p>
+                    </div>
+                    <div className={iconAndTextCardStyles.icon_and_text_card + ' ' + pageStyles.subsection_second_element_mobile_spacing}>
+                        <div className={iconAndTextCardStyles.icon}>
+                            <Image className={iconAndTextCardStyles.icon} src={SupportIcon} alt='icon'></Image>
+                        </div>
+                        <h1>
+                            Personal <br/> Support
+                        </h1>
+                        <p>
+                            The Mito team is around to help train, debug, or ideate on new features. 
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            <section>
+                <div className={pageStyles.subsection + ' ' + pageStyles.subsection_column + ' center'}>
+                    <h1>
+                        Make your team Python Self-Sufficient
+                    </h1>
+                    <p className='display-desktop-only-inline-block'>
+                        Leverage your team’s Excel expertise to make them Python self-sufficient. <br /> 
+                        Don’t throw away years of Excel skill building.
+                    </p>
+                </div>
+                <h2 className='margin-top-4rem margin-bottom-2rem'>
+                    Python Before Mito
+                </h2>
+                <div className={pageStyles.subsection + ' ' + teamsStyles.calendar_section + ' flex-row-desktop-only'}>
+                    <CalendarDay> 
+                        <h2>Mon</h2>
```

Comment:
Is there a particular reason these are short? I tried making them the full name, and it still works - let's do that?

Reply:
If we do the full words then when the screen gets smaller there isn't enough space for all of it, especially with the 4 cards.